pkg/filter/driftignore.go

```diff
@@ -47,7 +48,8 @@ func (r *DriftIgnore) readIgnoreFile() error {
 		nbArgs := len(typeVal)
 		if nbArgs < 2 {
 			logrus.WithFields(logrus.Fields{
-				"line": line,
+				"line":    strconv.Itoa(index + 1),
```

Comment:
You can start your index at 1 instead of 0 `index := 1`, and thus there you can remove the index+1. And you can rename index to `lineNumber` it will add semantic sense to this variable.

Maybe it will work without the `Itoa` conversion, can you try ?

Reply:
I doesn't work without the itoa cause the range give a loop object and print (&int=0)
The lineNumber = 1 seems cool, i was just thinking, if reusing the index variable in the loop so there is a need to start at 0.

Reply:
> I doesn't work without the itoa cause the range give a loop object and print (&int=0)

I don't know what do you mean by a loop object, but ok if logrus print int values like this `Itoa` makes sense `&int=0`